libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -382,13 +382,22 @@ export interface IBucketClient {
   /**
    * Copy an object to a new location in the bucket. If the destination object
    * already exists, it will be overwritten. Returns once the copying is finished.
-   *
    * @param srcKey The key of the source object you wish to copy.
    * @param dstKey The key of the destination object after copying.
    * @throws if `srcKey` object doesn't exist.
    * @inflight
    */
   copy(srcKey: string, dstKey: string): Promise<void>;
+
+  /**
+   * Move an object to a new location in the bucket. If the destination object
+   * already exists, it will be overwritten. Returns once the renaming is finished.
```

Comment:
`Returns once the renaming is finished.` feels a little bit confusing for me (as the method returns `Promise<void>`)
I see this line appears on the copy method too, @Chriscbr what do you think?

Reply:
(Since Chris is on vacation) 
@staycoolcall911 what do you think?

Reply:
I'd delete those 6 words altogether (also from `copy()`) - users of this API can see the `Promise` return type on the method signature and this text doesn't add any information.